src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -258,6 +258,10 @@ public ElementTag createOption(String text, String value, boolean isSelected) {
         }
     }
     
+    public static ElementTag createOption(String text, String value) {
+        return new ElementTag(text, "value", value);
+    }
```

Comment:
May be a good idea to put a method comment for these `createOption()` methods, mentioning that it's for the HTML `<option>` tag, rather than some generic option.